### PR TITLE
Development Process Page Add

### PR DESCRIPTION
**Pull request by ServiceDog**
Additions:
- Development Guide Page (under "Contribute")
- Area for Style Guides (linked to in the Development Guide)

Modifications:
- Added emotes to several pages
    - image
- Moved Contributor Introductions Page instructions to Contributor Hub
- Moved notes about the arrow token to the Contributor Hub, out of the intro.md
- Page Order
    - image

### CODE REVIEW

File: docs/contributing/intro.md [hunk 1]
```diff
@@ -4,20 +4,12 @@ sidebar_position: 1
...
 # Getting Started
 
-The best way to get connected with the Arrow community and start working is to join our Discord server. Introduce yourself in the "introductions" channel and we'll all be happy to get you up to speed - we're a friendly bunch!
+The best way to get connected with the Arrow community and start working is to join our Discord server!
 
-As a general rule of culture, you don't need formal permission to start working on anything, but it's best to communicate well with the community to keep us all aligned and working on high-impact tasks. 
+Introduce yourself in the `#introductions` channel and we'll all be happy to get you up to speed - we're a friendly bunch!
 
-As of now, we're still finalizing the details of our Arrow token, but we will make sure that all contributors are retroactively rewarded for all of their hard work. We strive to have a culture of merit - If you contribute something valuable, you will be rewarded.
+:warning: As a general rule of culture, you don't need formal permission to start working on anything. However, it's best to communicate well with the community to keep us all aligned and working on high-impact tasks.
```
Comment (GoodluckH) [inline docs/contributing/intro.md:11]: This doesn't render well. I think the emoji intended is the yellow triangle with exclamation mark? That emoji sometimes only renders as a small black-and-white symbol. 
Screen Shot 2022-07-08 at 11 31 00 AM

Suggest to replace with ❗️

File: docs/contributing/styleguides/_category_.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+    "label": "Code Style Guides",
```
Comment (GoodluckH) [inline docs/contributing/styleguides/_category_.json:2]: Do we want to push this to production right now if it's coming soon?

Comment (ServiceDog) [reply docs/contributing/styleguides/_category_.json:2]: Just wanted something to link to that wasn't broken, the style guides should come in the next few days for Rust and Typescript

Comment (GoodluckH) [reply docs/contributing/styleguides/_category_.json:2]: Sounds good.

File: docs/contributing/intro.md (hunk 1 above)
Comment (ServiceDog) [reply docs/contributing/intro.md]: Fixed in d84a92a

**Review by GoodluckH — APPROVED** [review]
LGTM

**Pull request merged**